Library/Formula/rssh.rb

```diff
@@ -11,11 +13,29 @@ class Rssh < Formula
     sha256 "abd625a8dc24f3089b177fd0318ffc1cf4fcb08d0c149191bb45943ad55f6934"
   end
 
+  # Submitted upstream:
+  # https://sourceforge.net/p/rssh/mailman/message/34917198/
+  if build.with? "zfs"
+    patch do
+      url "https://gist.github.com/kaazoo/1358e893033f536802f9/raw/e2bec9c1608ba18977bbea82a1ef8329dc55904d/rssh-2.3.4_zfs_support.patch"
+      sha256 "ea843389e402d79e2d62fb37da2c3bc3d16ead0cd84bc28824e2133635159603"
+    end
+  end
+
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+      --mandir=#{man}
+    ]
+    if build.with? "zfs"
+      # ZFS from https://openzfsonosx.org/ provides /usr/local/bin/zfs
+      args << "--with-zfs=/usr/local/bin/zfs"
```

Comment:
This path is only correct currently for the 10.11 installer. I think it might be better to add an actual `:zfs` `Requirement` to Homebrew or at the bare minimum call `which()`. If necessary I will consider installing a package config file instead, I guess, but that would not be my preference.

@mikemcquaid @DomT4 What do you prefer?